app/connector/SubmissionDraftConnector.scala

```diff
@@ -16,12 +16,14 @@
 
 package connector
 
+import akka.http.javadsl.model.ContentType
```

Comment:
Somehow this doesn't seem right. But I could be wrong. :)

Reply:
ahh yes! :D